glitter/pages/templates/admin/pages/page/duplicate_page.html

```diff
@@ -0,0 +1,82 @@
+{% extends "admin/base_site.html" %}
```

Comment:
Do we need this template, or could we just extend from `admin/change_form.html`?